### ISSUE TITLE
Add asciinema demo to README, documentation and website

### ISSUE THREAD

**Issue by romac** [epfl-lara/stainless#606]
https://asciinema.org

**Comment by vkuncak** [comment]
I thought someone had hijacked your account. The replay also needs a dedicated player apparently?

**Comment by romac** [comment]
Yes, so it would just link the image to a page on the asciinema website.

I could alternatively record it my terminal window, convert the video to a GIF and then inline it directly into the README, but that might make the page much heavier if we want to preserve legibility.

**Comment by vkuncak** [comment]
I would prefer not to have things start moving when one looks at the README file. People today get more than enough of moving ads; having some static oasis things where nothing jumps at you without you doing something first is a feature. :)
